packages/config/test/index.ts

```diff
@@ -180,40 +171,34 @@ test('when using --global, link-workspace-packages, shared-workspace-shrinwrap a
         version: '1.0.0',
       },
     })
-    t.notOk(config.linkWorkspacePackages, 'link-workspace-packages is false')
-    t.notOk(config.sharedWorkspaceLockfile, 'shared-workspace-lockfile is false')
-    t.notOk(config.lockfileDir, 'lockfile-dir is null')
+    expect(config.linkWorkspacePackages).toBeFalsy()
+    expect(config.sharedWorkspaceLockfile).toBeFalsy()
+    // FIXME: it supposed to return null but is undefined
+    expect(config.lockfileDir).toBeUndefined()
```

Comment:
I've noticed here the comment expect `null` but the outcome is clearly `undefined`.